packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3505,6 +3519,41 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.members.admitMembers(memberIds, locusUrls);
   }
 
+  /**
+   * Manages be right back status updates for the current participant.
+   *
+   * @param {boolean} enabled - Indicates whether the user enabled brb or not.
+   * @returns {Promise<void>} resolves when the brb status is updated or does nothing if not in a multistream meeting.
+   * @throws {Error} - Throws an error if the request fails.
+   */
+  // eslint-disable-next-line consistent-return
+  public async beRightBack(enabled: boolean): Promise<void> {
+    // this logic should be applied only to multistream meetings
+    if (!this.isMultistream) {
+      LoggerProxy.logger.error('Meeting:index#beRightBack --> Not a multistream meeting');
+    } else if (!this.mediaProperties.webrtcMediaConnection) {
+      LoggerProxy.logger.error(
+        'Meeting:index#beRightBack --> WebRTC media connection is not defined'
+      );
+    } else {
+      return this.meetingRequest
+        .sendBrb({
+          enabled,
+          locusUrl: this.locusUrl,
+          deviceUrl: this.deviceUrl,
+          selfId: this.selfId,
+        })
+        .then(() => {
+          this.sendSlotManager.setSourceStateOverride(MediaType.VideoMain, enabled ? 'away' : null);
+        })
+        .catch((error) => {
+          LoggerProxy.logger.error('Meeting:index#beRightBack --> Error ', error);
+
+          return Promise.reject(error);
+        });
+    }
+  }
```

Comment:
```suggestion
  public async beRightBack(enabled: boolean): Promise<void> {
    // this logic should be applied only to multistream meetings
    if (!this.isMultistream) {
      LoggerProxy.logger.error('Meeting:index#beRightBack --> Not a multistream meeting');
      return;
    } else if (!this.mediaProperties.webrtcMediaConnection) {
      LoggerProxy.logger.error(
        'Meeting:index#beRightBack --> WebRTC media connection is not defined'
      );
      return;
    }

    return this.meetingRequest
      .sendBrb({
        enabled,
        locusUrl: this.locusUrl,
        deviceUrl: this.deviceUrl,
        selfId: this.selfId,
      })
      .then(() => {
        this.sendSlotManager.setSourceStateOverride(MediaType.VideoMain, enabled ? 'away' : null);
      })
      .catch((error) => {
        LoggerProxy.logger.error('Meeting:index#beRightBack --> Error ', error);
        return Promise.reject(error);
      });
  }
```

Then i think something like this should fix it

Reply:
@SomeBody16 thanks! I will update this part, align with other public methods